tests/positive/files/file.go

```diff
@@ -129,7 +129,7 @@ func UserGroupByName() types.Test {
 			Contents: "example file\n",
 		},
 	})
-	configMinVersion := "2.1.0"
+	configMinVersion := "2.0.0"
```

Comment:
2.0.0 does not have the name key for users/groups

Reply:
Fixed